packages/spanner/system-test/spanner.js

```diff
@@ -108,6 +108,22 @@ var spanner = new Spanner(env);
       }, execAfterOperationComplete(done));
     });
 
+    describe('uneven rows', function() {
+      it('should allow inserting uneven rows', function(done) {
+        table.insert([
+          {
+            Key: generateName('id'),
+            BoolValue: true
+          },
+          {
+            Key: generateName('id'),
+            BoolValue: true,
+            IntValue: 10
+          }
+        ], done);
```

Comment:
These semantics of automatically setting unset keys to null is very confusing in the case of updates.
If in the above code if you had `table.update` instead of `table.insert`, it would be reasonable to expect that to leave IntValue untouched in the first row but that is not what it will do.